lib/crashreporting/data/bloc.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/foundation.dart';
```

Comment:
Probably don't need that here. For the `@immutable` annotation, the `package:meta/meta.dart` is sufficient.